template/condo.ps1

```diff
@@ -148,7 +148,9 @@ function Write-Info([string] $message) {
 
 function Get-File([string] $url, [string] $path, [int] $retries = 5) {
     try {
+        [Net.ServicePointManager]::SecurityProtocol = "tls12, tls11, tls"
```

Comment:
can you replace this entire file with the one I gave you? it adds a for-loop to make the retries actually work ;)

Reply:
fiiiiine